libs/books/feature/src/lib/reading-list/reading-list.component.html

```diff
@@ -12,9 +12,10 @@
     <div>
       <button
         mat-icon-button
+        test-id="remove-from-reading-list"
         color="warn"
         [attr.aria-label]="'Remove ' + b.title + ' from reading list'"
-        (click)="removeFromReadingList(b)"
+        (click)="removeFromReadingList(b);showSnackBar(b)"
```

Comment:
Is this really the right place to do this? What if `removeFromReadingList` fails?

I don't generally recommend attaching multiple functions to a `click` handler like this either.